lib/collection/benches/prof.rs

```diff
@@ -0,0 +1,89 @@
+use std::fs::File;
```

Comment:
Is it possible to share a single `prof.rs` file between all the `benches` across crates?